docs/cel2/operators/migrate-node.md

```diff
@@ -1,149 +1,146 @@
 
-## Migration overview
+If you would like to use `full` sync but don't want to migrate your own pre-hardfork datadir, you can also download a migrated datadir from the links provided in the [Network Config & Assets](run-node.md#network-config--assets) section. Note that migrated datadirs won't be available until after the hardfork. For minimal downtime, we recommend migrating your own data for the hardfork.
 
-In the Celo L1 to L2 transition, Celo L1 nodes will migrate all historical Celo
-data into the Celo L2 node, ensuring that blocks, transactions, logs, and
-receipts are fully accessible within the Celo L2 environment.
+:::note
+The terms L1 and pre-hardfork are used interchangeably to reference Celo before the L2 transition. L1 does not refer to Ethereum in this document.
+:::
 
-This process involves 4 steps:
+## Migration Overview
 
-1. Upgrading the L1 node to the latest client release so it stops producing blocks
-   at the hardfork
-2. Waiting for the hardfork
-3. Pulling the new network configuration files once they are known (e.g. hardfork block time)
-4. Launching the L2 node with snap sync
+Migrating a pre-hardfork datadir involves these high-level steps:
 
-Optionally, for node operators that want to verify the entire chain history
-through the upgrade, they may manually migrate their L1 (full) node chain data into a
-format compatible with the L2 node. This is required for all nodes looking to full
-sync and for anyone looking to run an archive node. Doing
-so involves these steps:
+1. Upgrade your L1 node to the [latest client release](run-node.md#network-config--assets) so it will stop producing blocks at
+   at the hardfork.
+2. 1-2 days before the hardfork, stop your node and run a pre-migration to migrate the majority of data. This is not required, but is highly recommended for minimizing downtime. See [Preparing for the L2 Hardfork](../notices/l2-migration.md).
+3. Restart your node and wait for the hardfork.
+4. Shut down your node once the hardfork block number is reached.
+5. Pull the necessary network configuration artifacts once available (e.g. hardfork block time).
+6. Run the migration tool to migrate your L1 datadir and produce the hardfork block.
+7. Launch your L2 node with the migrated datadir.
 
-1. Upgrading the L1 node to the latest client release so it stops producing blocks
-   at the hardfork
-2. Optionally running pre-migrations of the L1 datadir (see note below)
-3. Waiting for the hardfork
-4. Shutting down the L1 node
-5. Pulling the new network configuration files once they are known (e.g. hardfork block time)
-6. Running the migration tool to migrate the L1 datadir and produce the hardfork block
-7. Launching the L2 node with the migrated datadir
+The migration tool can be run multiple times as the L1 chain data grows and will continue migrating from where it last left off.
+Please note that the node must be stopped before the migration tool is run, even once it has reached the hardfork.
```

Comment:
nit: new line to make it stand out (and because there's no link to the previous sentence)